upgrade_test.py

```diff
@@ -847,6 +847,20 @@ def test_generic_cluster_upgrade(self):
 
         For multi-dc upgrades, alternates upgraded nodes between dc's.
         """
+        # In case the target version >= 3.1 we need to perform test for truncate entries
+        target_upgrade_version = self.params.get('target_upgrade_version').replace('~', '-')
+        self.truncate_entries_flag = False
+        if target_upgrade_version and parse_scylla_version(target_upgrade_version) >= parse_scylla_version('3.1') and \
+                not is_enterprise(target_upgrade_version):
+            self.truncate_entries_flag = True
```

Comment:
Where is this coming from?
Scylla 3.1 is very very old and we should remove from the code any checks for it.
If we have this flag we can either remove it (better) or set it to default (only if it's really necessary to control it).

Reply:
The only test that sets `truncate_entries_flag` to false and sticks with it is `test_upgrade_cql_queries`, which is not used at all whatsoever. So I'll remove it altogether.

Reply:
There's also the `is_enterprise` check which I don't see the purpose of? I don't see why should enterprise scylla be excluded from this check.
I see that it was added in https://github.com/scylladb/scylla-cluster-tests/pull/1191. @fruch I know this is pretty ancient stuff but is there a chance you remember why the check was added?

Reply:
it could be because enterprise was based on much older OSS version, or because the version verification would be broken, as every 20xx version will be bigger than 3.1...
i think we could remove it (and test if it works fine on enterprise)

Reply:
Remove this flag. 3.1 is ancient and we don't need it.

Reply:
@roydahan , the question was about checking `if is_enterprise()`